inc/class-group.php

```diff
@@ -83,6 +94,7 @@ public function get_page( $id ) : ?Page {
 		$current_path = $id;
 		// Crawl through all the url parts (seperated by /) to get the
 		// subpage from the parent at each step.
+		// phpcs:ignore WordPress.CodeAnalysis.AssignmentInCondition.FoundInWhileCondition
 		while ( $subpage_id = array_shift( $parts ) ) {
```

Comment:
Could instead change to:

```suggestion
		while ( count( $parts ) > 0 ) {
			$subpage_id = array_shift( $parts )
```

Reply:
Your code 😛

Reply:
live laugh learn